src/app/static/src/app/store/baseline/mutations.ts

```diff
@@ -9,16 +9,31 @@ interface BaselineMutation extends Mutation<BaselineState> {
 export interface BaselineMutations {
     PJNZLoaded: BaselineMutation
     PJNZUploadError: BaselineMutation
+    BaselineDataLoaded: BaselineMutation
 }
 
 export const mutations: MutationTree<BaselineState> & BaselineMutations = {
     PJNZLoaded(state: BaselineState, action: PJNZLoaded) {
         state.pjnzError = "";
-        state.country = action.payload.country
+        state.country = action.payload.country;
+        // TODO this step isn't really complete until all files are uploaded
+        // but for now lets say it is
+        state.complete = true;
     },
 
     PJNZUploadError(state: BaselineState, action: PJNZUploadError) {
         state.pjnzError = action.payload;
+    },
+
+    BaselineDataLoaded(state: BaselineState, action: BaselineDataLoaded) {
```

Comment:
So this BaselineDataLoaded mutation is the one that's triggered when we do a fetch of data from the server on page load, as opposed to PJNZLoaded which happens when a single file is uploaded? Would it be worth having a slightly different naming convention for those two types of mutation just to make it obvious what happened? Could leave this as is but rename PJNZLoaded to PJNZUploaded (mirroring PJNZUploadError).

Reply:
I think this change is from the previous branch that already got merged but yes, that's right. I see why this is confusing, agree should be PJNZUploaded.